src/lib/test-outbound/outbound-initiator.js

```diff
@@ -183,36 +183,20 @@ const terminateOutbound = (traceID) => {
 
 const processTestCase = async (testCase, traceID, inputValues, variableData, dfspId, globalConfig) => {
   const tracing = getTracing(traceID)
+  const apiDefinitions = openApiDefinitionsModel.getApiDefinitions()
 
-  // Load the requests array into an object by the request id to access a particular object faster
-  const requestsObj = {}
```

Comment:
The other part of the code is using requestsObj for quick lookup based on the id instead of calling find function on request array every time. Please observe the line number 221

Reply:
I assume you mean line 207 in the original code. Note that the proposed change doesn't perform any of this preamble, therefore it allocates less memory, does less work, and also does not perform an array find in the loop.

Reply:
I am referring to this line. This line is accessing requestsObj and the requestsObj is removed in the proposed changes.
`convertedRequest = replaceVariables(request, inputValues, request, requestsObj)`

Reply:
Ah, gotcha, I thought in your original message you were explaining how the code worked, not pointing me to a variable I'd removed that was still in use. I'll address this in the next day or two- thanks.

Reply:
I replaced the `requestsObj` object with an array find. I had a quick look at the collections in examples. For such small arrays, an array find is likely to be quicker than an object lookup. In my opinion it has the added bonus of increasing the readability of the code.